pyccel/parser/syntactic.py

```diff
@@ -630,8 +630,15 @@ def _visit_FunctionDef(self, stmt):
         if 'stack_array' in decorators:
             args = list(decorators['stack_array'].args)
             for i, arg in enumerate(args):
-                args[i] = str(arg).replace("'", '')
+                args[i] = str(arg)
             decorators['stack_array'] = tuple(args)
```

Comment:
If there is no need to perform any substitution in the arguments' names, we should rewrite this as
```python
decorators['stack_array'] = tuple(str(a) for a in decorators['stack_array'].args)
```